app/views/admin/health/coordination_teams/index.html.haml

```diff
@@ -0,0 +1,10 @@
+- content_for :title,  'Coordination Teams'
+%h1= content_for :title
+
+= render 'menus/health_admin_tabs'
+
+- if @teams.blank?
+  = render 'new team'
```

Comment:
I'm getting a missing partial here, should be
```suggestion
  = render 'new_team'
```

Reply:
Also there are no labels for the inputs, which makes it hard to know what to do
<img width="823" alt="Screen Shot 2022-07-25 at 3 30 02 PM" src="https://user-images.githubusercontent.com/5333982/180859273-7475d5ec-fe95-43d2-a092-fa9f9c9a9c24.png">

Reply:
Something weird here -- thanks! I'll figure out what happened and clean that up.